NuGet.Config

```diff
@@ -27,6 +27,7 @@
     <add key="myget.org roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
     <add key="myget.org devcore" value="https://www.myget.org/F/vs-devcore/api/v3/index.json" />
     <add key="myget.org roslyn-build-tools" value = "https://dotnet.myget.org/F/roslyn-build-tools/api/v3/index.json" />
+    <add key="myget.org roslyn-tools" value = "https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
```

Comment:
Hmm, I hadn't realized there was also a `roslyn-build-tools` feed. We should probably standardize on using just one of these.